src/ApiCore/LongRunning/Gapic/OperationsGapicClient.php

```diff
@@ -243,19 +247,21 @@ public function __construct($options = [])
 
         $this->scopes = $options['scopes'];
 
-        $createStubOptions = [];
-        if (array_key_exists('sslCreds', $options)) {
-            $createStubOptions['sslCreds'] = $options['sslCreds'];
+        if (empty($options['createTransportFunction'])) {
+            $options['createTransportFunction'] = function ($options, $transport = null) {
+                switch ($transport) {
+                    case 'grpc':
+                        if (empty($options['createGrpcStubFunction'])) {
+                            $options['createGrpcStubFunction'] = function ($fullAddress, $stubOpts, $channel) {
```

Comment:
If we move setting the default value of `$options['createGrpcStubFunction']` up to where the rest of the default options are set, then I think all of the rest of this function (including the switch case) can be pushed into the call to `$this->getTransport($options)`. Does that work?

Reply:
This is outdated, and will be updated once I have regenerated the GAPICs.

Reply:
we have removed this all together. We're just waiting on the generation of the GAPIC clients to remove this.